test_conformance/pipes/test_pipe_info.cpp

```diff
@@ -40,7 +41,7 @@ int test_pipe_info( cl_device_id deviceID, cl_context context, cl_command_queue
     if (pipe_width != returnVal)
     {
         log_error("Error in clGetPipeInfo() check of pipe packet size\n");
-        return -1;
+        return TEST_FAIL;
```

Comment:
test_fail() includes this functionality (here and 2 places below)

Reply:
I'm not sure that the macros are named in the best way, but I've rewritten the code to use these macros.  Thanks for pointing this out!